docs/ci-setup.md

```diff
@@ -0,0 +1,20 @@
+# CI Setup
+
+## Pipeline
+
+1. unit-tests: fast jobs for fast feedback (linting, etc...)
```

Comment:
I could not see actual CI jobs for each item on this list.
Actually I could not find the item for Terraform jobs(`Kubespray CI - tf-*`).
How about adding actual CI job name as a sample for each item for easy understanding?